Cargo.toml

```diff
@@ -1,24 +1,27 @@
 [package]
 authors = ["omnect@conplement.de>"]
-edition = "2018"
+description = "This experimental service allows wifi commissioning via bluetooth."
+edition = "2021"
+homepage = "https://www.omnect.io/home"
 license = "MIT OR Apache-2.0"
 name = "wifi-commissioning-gatt-service"
 repository = "git@github.com:omnect/wifi-commissioning-gatt-service.git"
-version = "0.4.1"
+version = "0.4.2"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 async-trait = "0.1"
-bluer = { version = "0.15", features = ["bluetoothd"] }
-clap = { version = "3.2", features = ["derive"] }
+bluer = { version = "0.17", features = ["bluetoothd"] }
+clap = { version = "4.5", features = ["derive"] }
 enclose = "1.1"
-env_logger = "0.8"
+# ToDo: check if 'default-features = false' can be removed. currently needed as workaround for cargo-bitbake.
+env_logger = { version = "0.11", default-features = false }
```

Comment:
what does it mean? which functionality changed?

Reply:
we want to update env_logger in order to get rid of `atty` audit warning.

if we use 0.11 with default features I get the following:
```
jan@JZ-Mint-I631:~/projects/wifi-commissioning-gatt-service$ cargo-bitbake bitbake
error: failed to select a version for `env_logger`.
    ... required by package `wifi-commissioning-gatt-service v0.4.2 (/home/jan/projects/wifi-commissioning-gatt-service)`
versions that meet the requirements `^0.11` are: 0.11.3

the package `wifi-commissioning-gatt-service` depends on `env_logger`, with features: `anstream` but `env_logger` does not have these features.
 It has an optional dependency with that name, but but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.


failed to select a version for `env_logger` which could resolve this conflict
```

Reply:
which functionality changes when we use the crate without default features? if nothing, it seems we don't need this default features anyway.

Reply:
https://github.com/rust-cli/env_logger/blob/0e25d9e993241f1c57f6f7c2ab775a407a87fa3f/Cargo.toml#L119C1-L119C47

at least it still compiles. so yes I guess we don't need the features at all